extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -107,40 +103,35 @@
 }
 
 .i-amphtml-story-button {
-  background-repeat: no-repeat !important;
-  background-position: center center !important;
   height: 48px !important;
   width: 48px !important;
   cursor: pointer !important;
   border: none !important;
   pointer-events: auto !important;
-  /* for svg backgrounds: */
-  background: center center no-repeat !important;
-  background-size: 24px 24px !important;
+  background: none !important;
 }
 
 [desktop] .i-amphtml-story-button {
   margin: 0 4px !important;
-  background-size: 32px 32px !important;
-  padding: 8px !important;
 }
 
 .i-amphtml-story-button:active {
-  background-color: rgba(0, 0, 0, 0.2) !important;
+  background-color: #0003 !important;
```

Comment:
Are we sure we want to remove the opacity and completely hide the background?

Reply:
These two colors are very similar, `#rgba` is the short hand for `#rrggbbaa` so the `0x3` at the end is the alpha value. We might change the active state of the buttons to sth more fancy when we have more time to design animations.